packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -1,7 +1,144 @@
+import React from 'react';
+import { withBaseElementProps } from '@aws-amplify/ui-react-core/elements';
+
 import { StorageBrowserElements } from '../../context/elements';
 
-export interface TableControl<T extends StorageBrowserElements>
-  extends Pick<
+const {
+  Table: BaseTable,
+  TableBody: BaseTableBody,
+  TableData: BaseTableData,
+  TableHead: BaseTableHead,
+  TableHeader: BaseTableHeader,
+  TableRow: BaseTableRow,
```

Comment:
Should I add other elements like `caption`?

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption